packages/sections/src/evidence/OTGenetics/Body.jsx

```diff
@@ -23,200 +25,217 @@ import OPEN_TARGETS_GENETICS_QUERY from "./sectionQuery.gql";
 import { otgStudyUrl, otgVariantUrl } from "../../utils/urls";
 import { identifiersOrgLink, sentenceCase } from "../../utils/global";
 
-function getColumns(label) {
-  return [
-    {
-      id: "disease",
-      label: "Disease/phenotype",
-      renderCell: ({ disease }) => <Link to={`/disease/${disease.id}`}>{disease.name}</Link>,
-      filterValue: ({ disease }) => disease.name,
-    },
-    {
-      id: "diseaseFromSource",
-      label: "Reported disease/phenotype",
-      renderCell: ({ diseaseFromSource, studyId }) => {
-        const parsedDiseaseFromSource = diseaseFromSource.replace(/['"]+/g, "");
-        return (
-          <Link external to={otgStudyUrl(studyId)}>
-            {diseaseFromSource ? parsedDiseaseFromSource : studyId}
-          </Link>
-        );
+function Body({ id, label, entity }) {
+  const { ensgId, efoId } = id;
+  const variables = { ensemblId: ensgId, efoId, size: sectionsBaseSizeQuery };
+
+  function getColumns(label) {
```

Comment:
Please keep the `getColumns` function out of the scope of the **Body** component. This will keep it accessible for unit testing. You can pass params as props and/or memorize the result of the function if needed